### PR TITLE
Updated Genius

### PR DESCRIPTION
**Pull request by fbernhart**
genius

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
It appears the link to the source is broken. Could you update to the [Genius website](), which includes the logo built into the header.

Might it also be worth re-orienting it so that it looks more like the 'G' shape? I'm not familiar with the brand, so not sure which method is preferred. @simple-icons/maintainers - any guidance here?

3 decimal path generated from header:
```svg
M22.678,2.361C22.678,2.361,22.678,2.164,22.678,2.361C21.104,0.787,19.333,0,17.169,0s-4.131,0.787-5.508,2.164 c-0.393,0.393-0.59,0.787-0.984,1.18v0.197c0,0.197,0.197,0.197,0.197,0.197c0.787-0.393,1.77-0.59,2.754-0.59 c2.164,0,4.328,0.984,5.705,2.361l3.148,0c0.197,0,0.197-0.197,0.197-0.197L22.678,2.361C22.874,2.361,22.678,2.361,22.678,2.361z M22.481,11.41l-2.951,0c-1.574,0-2.754-1.18-2.951-2.754c0.197,0,0-0.197,0-0.197c-0.59,0-1.18,0.393-1.574,0.787v0.393 c-1.18,3.541,0.197,4.721,1.77,4.721l2.164,0c0.197,0,0.197,0.197,0.197,0.197v0.787c0,0.197,0.197,0.197,0.197,0.197 c1.18-0.197,2.361-0.787,3.344-1.574v-2.361C22.874,11.607,22.678,11.41,22.481,11.41z M22.481,20.066 C22.481,20.066,22.284,20.066,22.481,20.066C22.284,20.066,22.284,20.066,22.481,20.066C22.284,20.066,22.284,20.066,22.481,20.066 c-1.574,0.59-3.148,0.984-4.918,0.984c-7.279,0-13.377-5.902-13.377-13.377c0-1.77,0.393-3.344,0.984-4.918 c0-0.197-0.197-0.197-0.393-0.197H4.579C2.415,4.918,1.235,7.869,1.235,11.41C1.235,18.492,6.94,24,13.825,24 c3.344,0,6.492-1.377,8.656-3.541v-0.197C22.678,20.262,22.481,20.066,22.481,20.066z
```

**Comment by ericcornelissen** [comment]
I'm not terribly familiar with the brand either, but whenever I see just the "G" it is at a 90 degree angle. See for example the website favicon or the app icon in the Play Store or App store.

File: icons/genius.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Genius icon</title><path d="M12.8967 1.2348c-.36.001-.722.013-1.08.017-.218-.028-.371.225-.352.416-.035 1.012.023 2.0251-.016 3.0362-.037.841-.5551 1.596-1.2241 2.08-.5.345-1.118.4351-1.6711.6631.121.78.434 1.556 1.057 2.0701 1.189 1.053 3.2242.86 4.1702-.426.9451-1.071.453-2.5731.603-3.8542.2861-.48.9371-.132 1.3171-.49-.34-1.2491-.81-2.5292-1.725-3.4722a11.1256 11.1256 0 00-1.0801-.04zm-10.4205.006C.529 2.9918-.387 5.797.153 8.3612c.384 2.052 1.6821 3.8932 3.4502 4.9972.134-.23.23-.476.09-.73-.95-2.8142-.138-6.1193 1.9861-8.1904.014-.9861.043-1.9762-.003-2.9612l-.188-.214c-1.003-.051-2.008 0-3.0101-.022zm17.8809.055l-.205.356c.265.938.6 1.8621.72 2.8342.58 3.5461-.402 7.3133-2.6141 10.1405-1.8161 2.3531-4.4412 4.0742-7.3344 4.7732-2.6601.66-5.5143.45-8.0644-.543-.068.079-.207.237-.275.318 2.6641 2.6292 6.5433 3.9692 10.2595 3.4982 3.0752-.327 5.9953-1.8651 8.0234-4.1952 1.9351-2.1871 3.0832-5.0703 3.1252-7.9924.122-3.3842-1.207-6.8194-3.6362-9.1905z"/></svg>
```
Comment (adamrusted) [inline icons/genius.svg:1]: Can this be reduced to 3 decimals at all? 😅

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by fbernhart** [comment]
@adamrusted Done

**Review by adamrusted — APPROVED** [review]
Amazing! Thanks @fbernhart 🎉

**Pull request merged**